recipes/sqloxide/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "sqloxide" %}
+{% set version = "0.1.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wseaton/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5e46a472305d5a5ad1645ff930dc7d8da92563ea5804622c16d11db85f54d4f9
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+  host:
+    - python
+    - pip
+    - poetry-core >=1.0.0
+    - setuptools >=41.0.0
+    - wheel
+    - setuptools-rust >=0.10.2
+  run:
+    - python
+    - libgcc-ng  # [linux]
+    - compilers
```

Comment:
maybe this looks very weird here .. but without that, the packaging didn't work (see checks logs)
not sure if would have a better solution for that